src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped work");

```

Comment:
Similarly, the message 'Excavator stopped work' could be more informative. It should describe the state of the Excavator when it stops working, perhaps indicating that it is shutting down or that it has completed its tasks.